README.md

```diff
@@ -105,7 +105,9 @@ only when a rule or set of rules explicitly or implicitly permits it.
 Mutable Rules 
 -------------
 
-**201** Players may take their turn at any time, independent of other players. 
+**303** Players may take a turn at any time, independent of other players.
+Players are allowed to take any number of turns at the same time, regardless
+of any other ongoing turns taken by them.
 All players begin with zero points.
```

Comment:
Your original text does indeed seem to imply that these 'multiple turns' should be submitted as a batch. What about:

A player may have an infinite number of concurrent turns.
A player may start a turn regardless of the state of their other turns.